utils.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
@BenjaminTrapani next time please be more careful of your tone in emails, or in ticketing systems that generate emails. I understand you were frustrated but email is rarely the right time to vent.

In general the expectation for reviewers is that they understand the feedback before acting on it to avoid this scenario. This isn't limited to code reviews either but applies to any review - JIRA tickets, docs, etc. If you're not 100% certain you understand something please ask questions. Your comment suggests an awareness of how best to optimize your time. clypd benefits the most when you learn something since that helps you be more efficient the next time you encounter a similar problem and you'll learn best by understanding the feedback.

In this case I could have provided more details, "package name seems wrong here. it's uncommon (and broken, I think) to include a 'main' package alongside a non-main. I'm not even sure that the Go tools support this so there's something weird going on here." As someone who writes many reviews I try to adjust the details to match the context - the complexity of the PR and related code, the person being reviewed, etc. It's easy to make a mistake which is why it's so important for reviewees to ask clarifying questions.